src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetDelegateForFunctionPointerTests.cs

```diff
@@ -16,6 +16,7 @@ public class GetDelegateForFunctionPointerTests
         [InlineData(typeof(NonGenericDelegate))]
         [InlineData(typeof(MulticastDelegate))]
         [InlineData(typeof(OtherNonGenericDelegate))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/39187", TestPlatforms.Browser)]
```

Comment:
It looks like the only tests on this class that weren't marked as ActiveIssue are negative tests, i.e. ones validating an exception is thrown.  I'd suggest just putting the ActiveIssue on the whole test class instead.

Reply:
Done  :)